src/utils/theme.js

```diff
@@ -20,17 +23,18 @@ export let mergeNestedTheme = (theme, key) =>
   )(theme)
 
 let useTheme = (name, propTheme) =>
-  mergeOrReturn(
+  mergeOrReturnAll([
+    defaultTheme,
```

Comment:
We should not merge this in here. React will provide this by default if there is nothing on `Provider` - this would just defend against incomplete themes which we don't care about. Even worse, this will make so that we'll always be doing a merge on every render of every component because the default and context themes will most often always be present